frappe/email/__init__.py

```diff
@@ -19,11 +19,12 @@ def get_contact_list(txt, page_length=20) -> list[dict]:
 	reportview_conditions = build_match_conditions("Contact")
 	match_conditions = f"and {reportview_conditions}" if reportview_conditions else ""
 
+	# nosemgrep
```

Comment:
This isn't required. Please dont add it, the failure only lasts for one PR, we eventually need to remove all failures and when we scan for them in future we wont find it if `nosemgrep` is added. It's only required for false positives, actual failures should continue to fail. 

cc: @barredterra

Reply:
Thanks for the heads up. BTW, I did already refactor the query in #20323